### PR TITLE
update

### PR DESCRIPTION
**Pull request by vikas924**
Dear Reviewer,
Please review the changes made and the queries implemented.

Project Requirements
Created owners table with columns id, full_name, and age.
Created species table with columns id and name.
Modified animals table to include species_id and owner_id columns.
Inserted provided data into owners and species tables.
Updated animals with appropriate species_id and owner_id.
Implemented all changes in schema.sql and data.sql files.

Screenshots of Queries

- What animals belong to Melody Pond?
- List of all animals that are pokemon (their type is Pokemon).
- List all owners and their animals, including those without any animal.

- How many animals are there per species?
- List all Digimon owned by Jennifer Orwell.
- List all animals owned by Dean Winchester that haven't tried to escape.

- Who owns the most animals?

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -145,3 +145,55 @@ GROUP BY Species;
...
+GROUP BY species.name;
+-- DONE
+
+--   List all Digimon owned by Jennifer Orwell.
+SELECT animals.name, owners.full_name
+FROM animals
+INNER JOIN owners ON animals.owner_id = owners.id
+INNER JOIN species ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell';
+-- DONE
```
Comment (Strangeal) [inline queries.sql]: - [ ] Great work on your query! 👏 However, I noticed a small issue that needs attention. When I examined the data you inserted into the database, it appears that `Jennifer Orwell` owns two animals, but only one of them is of type `Digimon`. 

   - Your query is designed to retrieve all `Digimons` owned by `Jennifer Orwell`, but it's currently returning both of her animals, including the one that isn't a `Digimon`. To ensure that your query provides accurate results, I recommend making the necessary adjustments to filter out only the `Digimons` owned by `Jennifer Orwell`. 

Please don't forget to update your pull request summary with the screenshot of the revised query results. 🔄 
Happy coding! 😊🚀🍀

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @vikas924, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅ 
- Well structured project ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (vikas924) [reply queries.sql]: Made required changes

- List all Digimon owned by Jennifer Orwell.

Screenshot 2023-09-30 130345

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @vikas924 , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Good SQL queries
✅ ... Good pull request sumary

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**